app/src/main/java/org/fossasia/openevent/general/ticket/TicketDetailsViewHolder.kt

```diff
@@ -25,9 +25,14 @@ class TicketDetailsViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView
                 itemView.subTotal.text = "${eventCurrency}0.00"
             }
             TICKET_TYPE_PAID -> {
-                itemView.price.text = "$eventCurrency${"%.2f".format(ticket.price)}"
-                val subTotal: Float? = ticket.price.times(ticketQuantity)
-                itemView.subTotal.text = "$eventCurrency${"%.2f".format(subTotal)}"
+                if (ticket.price == 0f) {
+                    itemView.price.text = resource.getString(R.string.free)
+                    itemView.subTotal.text = "${eventCurrency}0.00"
+                } else {
```

Comment:
This shouldn't be needed

Reply:
So what should I do to solve this issue?

Reply:
> So what should I do to solve this issue?

Open an issue on the server-side to make ticket type free instead of rs 00 and let it be same.

Reply:
Make sure if the ticket type is free from the server-side response, it should display Free instead of Rs 00 in the app also.

Reply:
Ok.
Thanks @liveHarshit .